.eslintrc

```diff
@@ -20,7 +20,7 @@
   },
   "parser": "babel-eslint",
   "rules": {
-    "import/no-unresolved":false,
+    "import/no-unresolved":"off",
```

Comment:
Why do we need this off?

Reply:
It is the new syntax, you can choose either `off` or `0` https://eslint.org/docs/user-guide/configuring#configuring-rules